sdcm/nemesis.py

```diff
@@ -3976,8 +3976,15 @@ def start_and_interrupt_repair_streaming(self):
 
         self.target_node.wait_node_fully_start()
 
-        with adaptive_timeout(Operations.REBUILD, self.target_node, timeout=HOUR_IN_SEC * 48):
-            self.target_node.run_nodetool("rebuild", long_running=True, retry=0)
+        is_rebuild_supported = SkipPerIssues('scylladb/scylladb#17575', params=self.tester.params)
+        # If tablets in use and rebuild is not supported, running a DC repair instead.
+        with self.cluster.cql_connection_patient(self.target_node) as session:
+            if is_tablets_feature_enabled(session=session) and not is_rebuild_supported:
+                for node in [n for n in self.cluster.nodes if n.dc_idx == self.target_node.dc_idx]:
+                    node.run_nodetool(sub_cmd="repair")
```

Comment:
I would recommend doing long_running=True, retry=0

also maybe to consider hard timeout

Reply:
also I'm not sure you have guarantee all the nodes in this DC are up and running...